client/client.go

```diff
@@ -17,7 +17,16 @@ type Client struct {
 	endpoint string
 }
 
-type Toxic map[string]interface{}
+type Attributes map[string]interface{}
+
+type Toxic struct {
+	Name       string     `json:"name"`
+	Type       string     `json:"type"`
+	Stream     string     `json:"stream"`
```

Comment:
We can make use of `json:"stream,omitempty"` here for default values.
I'd like to continue allowing toxics to be of toxicity 0, at least in the server, but I'm not sure what to do about that on the client. Maybe we just don't have a "default" value in the client.